Source/WTF/wtf/Compiler.h

```diff
@@ -272,10 +272,11 @@
 
 // 32-bit platforms use different calling conventions, so a MUST_TAIL_CALL function
 // written for 64-bit may fail to tail call on 32-bit.
+// It also doesn't work on ppc64le: https://github.com/llvm/llvm-project/issues/98859
 #if COMPILER(CLANG)
 #if __SIZEOF_POINTER__ == 8
 #if !defined(MUST_TAIL_CALL) && defined(__cplusplus) && defined(__has_cpp_attribute)
-#if __has_cpp_attribute(clang::musttail)
+#if __has_cpp_attribute(clang::musttail) && !defined(__powerpc__)
```

Comment:
It seems that my submission with the Skia and skcms changes will land, see:

- https://skia-review.googlesource.com/c/skia/+/910216
- https://skia-review.googlesource.com/c/skcms/+/910656

...which means we will only need this change in `Compiler.h` from this PR in the end.

Reply:
Right. What we'll do here is land only the Compiler.h change via this merge request, and pick up the rest of the changes in our main branch whenever we next update skia.

I will separately land the full set of changes (sans the patching framework) on the webkitglib/2.46 branch.

Reply:
2.46: https://github.com/WebKit/WebKit/commit/33bea1079e80fcbff6208a6b52c027d00c23f95d